addons/website_slides/models/slide_slide.py

```diff
@@ -127,6 +127,7 @@ class Slide(models.Model):
     partner_ids = fields.Many2many('res.partner', 'slide_slide_partner', 'slide_id', 'partner_id',
                                    string='Subscribers', groups='base.group_website_publisher')
     slide_partner_ids = fields.One2many('slide.slide.partner', 'slide_id', string='Subscribers information', groups='base.group_website_publisher')
+    slide_partner_id = fields.Many2one('slide.slide.partner', string="Subscriber information", compute='_compute_slide_partner_id')
```

Comment:
help message or comment in compute method

Reply:
Help message is indeed nice to have.